### PR TITLE
[stable18] Add new recommended 18 apps

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

File: core/src/components/setup/RecommendedApps.vue [hunk 1]
```diff
@@ -82,6 +82,12 @@ const recommended = {
 	talk: {
 		description: t('core', 'Screensharing, online meetings and web conferencing – on desktop and with mobile apps.'),
 	},
+	onlyoffice: {
+		description: t('core', 'Collaboratively edit office documents'),
```
Comment (jancborchardt) [inline core/src/components/setup/RecommendedApps.vue]:
```suggestion
		description: t('core', 'Collaboratively edit office documents.'),
```

File: core/src/components/setup/RecommendedApps.vue [hunk 2]
```diff
@@ -82,6 +82,12 @@ const recommended = {
...
+	},
+	documentserver_community: {
+		description: t('core', 'Local document editing back-end used by the OnlyOffice app'),
```
Comment (jancborchardt) [inline core/src/components/setup/RecommendedApps.vue]:
```suggestion
		description: t('core', 'Local document editing back-end used by the OnlyOffice app.'),
```

**Review by jancborchardt — APPROVED** [review]
If you feel like it, add the full stop at the end as suggested, otherwise good to go! :)

**Pull request merged**